src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,243 @@
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_MCP_1_default_config(self, filename):
+    """ Generates the WinnForum configuration for MCP.1. """
+    # Load devices for SAS UUT for multiple iterations through multiple domain proxy's
+    device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    device_5 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    device_6 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_f.json')))
+
+    # Load Grant requests
+    grant_request_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_request_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+    grant_request_3 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+    grant_request_4 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_3.json')))
+    grant_request_5 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_request_6 = json.load(
+      open(os.path.join('testcases', 'testdata', 'grant_1.json')))
+  
+    # device_b is Category B
+    # Load Conditional Data
+    self.assertEqual(device_2['cbsdCategory'], 'B') 
+    conditionals_b = {
+        'cbsdCategory': device_2['cbsdCategory'],
+        'fccId': device_2['fccId'],
+        'cbsdSerialNumber': device_2['cbsdSerialNumber'],
+        'airInterface': device_2['airInterface'],
+        'installationParam': device_2['installationParam'],
+        'measCapability': device_2['measCapability']
+    }
+    
+    conditionals = {'registrationData': [conditionals_b]}
+    # Remove conditionals from registration
+    del device_2['cbsdCategory']
+    del device_2['airInterface']
+    del device_2['installationParam']
+    del device_2['measCapability']
+   
+    # Load GWPZ Record
+    gwpz_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+
+    # Load FSS record
+    fss_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+
+    # Load ESC record
+    esc_record_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+  
+    # Load PPA record
+    ppa_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'ppa_record_0.json'))) 
+    pal_record = json.load(
+      open(os.path.join('testcases', 'testdata', 'pal_record_0.json')))
+
+    pal_low_frequency = 3550000000.0
+    pal_high_frequency = 3560000000.0        
+
+    ppa_record_1, pal_record_1 = makePpaAndPalRecordsConsistent(ppa_record,
```

Comment:
Please use a plural variable name for the PAL record output since it is a list.

Reply:
Modified   to ppa_records_1 and pal_records_1